config.yaml

```diff
@@ -0,0 +1,194 @@
+  # special chars
+- from: KEY_E
+  to: KEY_ESC
+
+# alternative syntax
+- {from: KEY_D, to: KEY_DELETE}
+
+# possibly even
+- layer: KEY_SPACE
```

Comment:
i know this is a future suggestion only, but just a small comment that it would mean a lot of repetition compared to nested approach..